### PR TITLE
Implement PuppeteerLoaderFixture

### PR DESCRIPTION
**Pull request by kblok**
This class will help us download chromium and set up a WebServer on a CI server.

Wanna take a look @Meir017 ?

### CODE REVIEW

File: lib/PuppeteerSharp.Tests/PuppeteerLoaderFixture.cs [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            var downloaderTask = Downloader.CreateDefault().DownloadRevisionAsync(TestConstants.ChromiumRevision);
+            var serverTask = StartWebServerAsync();
+
+            await Task.WhenAll(downloaderTask, serverTask);
+
+        }
+
+        private async Task StartWebServerAsync()
+        {
+            var taskWrapper = new TaskCompletionSource<bool>();
```
Comment (Meir017) [inline lib/PuppeteerSharp.Tests/PuppeteerLoaderFixture.cs]: Would it be easier if we ran the server by adding a project-reference to `PuppeteerSharp.TestServer`?

Comment (kblok) [reply lib/PuppeteerSharp.Tests/PuppeteerLoaderFixture.cs]: I've tried that when I first started the Test project a few months ago. The thing is that `host.Run();` locks the thread and I wasn't able to bypass that.

Any ideas?

Comment (Meir017) [reply lib/PuppeteerSharp.Tests/PuppeteerLoaderFixture.cs]: https://github.com/aspnet/Hosting/blob/dev/src/Microsoft.AspNetCore.Hosting/WebHostExtensions.cs#L59.
could we just call `host.RunAsync` and save the returned `Task` then we could terminate the server by stoping the task, does this make sense?

Comment (kblok) [reply lib/PuppeteerSharp.Tests/PuppeteerLoaderFixture.cs]: I think I tried that, but I'll give it another shot!

Comment (Meir017) [reply lib/PuppeteerSharp.Tests/PuppeteerLoaderFixture.cs]: Maybe merge this and create a task to fix (if needed) it later in order to get the CI working on all branches?

**Comment by Meir017** [comment]
how is the CI configured? I don't see an `appveyor.yml` file?

File: lib/PuppeteerSharp.Tests/PuppeteerLoaderFixture.cs (hunk 1 above)
Comment (kblok) [reply lib/PuppeteerSharp.Tests/PuppeteerLoaderFixture.cs]: I tried RunAsync, the problem is that the same process can't run the tests and serve the page at the same time. I'll leave this at it is now, you can play with it if you want to.

**Comment by kblok** [comment]
@Meir017 It CI is configured in AppVeyor, I don't think we need the yml file (yet)

**Comment by Meir017** [comment]
I noticed that the ci runs are taking around 1 Hour to finish, could it be because the server process is still open? I'm guessing that the 1 Hour is the default test run timeout

**Comment by kblok** [comment]
I'll take a look

**Pull request merged**